### DISCUSSION TITLE
Gibt es Zeit-Budgets für Benutzer?

### DISCUSSION THREAD

**Discussion by julianlucks** [kimai/kimai#3682] [Questions & Answers]
Wir wollen Kimai zum Timetracking in unserer Firma nutzen. Wir arbeiten B2B, also haben wir verschiedene Kunden. Diesen Kunden können wir mit Kimai Zeit-Budgets zuweisen, was für uns zweckmäßig ist. 

Aber ich finde nirgends ein Feature für ein übergreifendes Stundenkonto, also um unseren Mitarbeitern / Benutzern ein (am besten wöchentliches) Zeit-Budget/Soll zuzuweisen für das Tracking von Überstunden etc.

Ist das nur über einen Workaround möglich, die eigene Firma als Kunden anzulegen und ein Zeit-Budget zu vergeben? Damit bliebe dann nichts anderes übrig als unsere Kunden als Projekte anzulegen und unsere Projekte als Tätigkeiten ... Gibt es vielleicht eine andere Lösung?

**Comment by kevinpapst** [comment]
Ich verstehe nicht wirklich was Du machen willst. 
Ich rate mal ... Kimai ist eine Projektzeiterfassung, momentan nicht gemacht zum Führen von Zeitarbeitskonten.
Man kann das aber nachrüsten, dafür könnt Ihr Euch das Controlling Plugin anschauen: 
https://www.kimai.org/store/mr-software-controlling-bundle.html (weitere Fragen dann bitte an den Entwickler, verlinkt auf der Seite)

**Comment by HeinzWuert** [comment]
Nur mal so quer gedacht: Man könnte ein Projekt mit dem Nutzernamen anlegen und diesem Projekt dann ein monatliches Zeitbudget zuordnen. 
Für diese Projekte dann spezielle Activities definieren für Arbeit, Krank und Urlaub.
Vielleicht will das mal jemand ausprobieren ;-)
